x-pack/plugins/apm/server/lib/connections/get_connection_stats/get_destination_map.ts

```diff
@@ -144,6 +145,7 @@ export const getDestinationMap = ({
           events: [ProcessorEvent.transaction],
         },
         body: {
+          track_total_hits: 10_000,
```

Comment:
Maybe use a constant instead of a literal? (Btw, i like the _ for readability)

Reply:
Looking closer, I'm not sure why I made it `10_000` and not just `false` in the first place.  Afaict we don't use the `hits.total` from these two requests.